api/envoy/extensions/filters/http/composite/v3/composite.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.composite.v3;
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.composite.v3";
+option java_outer_classname = "CompositeProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Composite]
+// Composite Filter :ref:`configuration overview <config_http_filters_cors>`.
+// [#extension: envoy.filters.http.composite]
+
+// Composite filter config.
+message Composite {
+  // List of permissible types that may be provided via the matching API.
+  repeated string type_url = 1;
+}
+
+message CompositeAction {
+  google.protobuf.Any typed_config = 1;
```

Comment:
This is a matcher action? Might be good to have some comments to that effect..

Reply:
Yea docs are lacking quite a bit, let me put something together

Reply:
Should this be a `TypedExtensionConfig`?

Reply:
Yea I think that makes more sense, I had it as Any since I could get everything to work with that but it seems more consistent to use a TEC here